components/sandbox/base/monitoring/sandbox-registration-service-proxy.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: toolchain-host-operator
+  name: sandbox-registration-service-proxy
+  labels:
+    prometheus: appstudio-workload
+spec:
+  endpoints:
+    interval: 15s
+    scheme: http
+    path: /metrics
+    port: proxy-metrics
```

This should be something like this I suspect:
```suggestion
  endpoints:
    - interval: 15s
      scheme: http
      path: /metrics
      port: proxy-metrics
```